images/capi/ansible/roles/providers/tasks/vmware-redhat.yml

```
@@ -21,7 +21,15 @@
     packages:
       - cloud-init
       - cloud-utils-growpart
+
+- name: Install python2 pip
+  yum:
```

To fix the lint issue:
```suggestion
  ansible.builtin.yum:
```